core-1/src/main/java/SpringApp.java

```diff
@@ -7,14 +7,21 @@
 
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
-import parkinglot.Car;
+import org.springframework.stereotype.Component;
+import scanning.parkinglot.ProjectConfig;
 
 public class SpringApp {
 
     public static void main(String[] args) {
         // initiate application context there
-        ApplicationContext context = new AnnotationConfigApplicationContext("parkinglot/beans");
+        ApplicationContext context = new AnnotationConfigApplicationContext(ProjectConfig.class);
         //print all created bean names
-        context.getBeansOfType(Car.class).forEach((s, car) -> car.startEngine());
+        //context.getBeansOfType(Car.class).forEach((s, car) -> car.startEngine());
+
+        String[] beanNames = context.getBeanNamesForAnnotation(Component.class);
+
+        for (String beanName : beanNames) {
+            System.out.println(beanName);
```

Comment:
I see the output here: 

projectConfig
fourCylinderEngine
volkswagenBeetle

these are not proper bean names, the output should be "oldSchoolStuff" and "Bumblebee"